### ISSUE TITLE
Report  Unexpected end of JSON input

### ISSUE THREAD

**Issue by talktosalvador** [vknabel/vscode-swiftlint#53]
```
SyntaxError: Unexpected end of JSON input
	at JSON.parse (<anonymous>)
	at /Users/svip/.vscode/extensions/vknabel.vscode-swiftlint-1.7.2/out/lint.js:137:30
	at Generator.next (<anonymous>)
	at fulfilled (/Users/svip/.vscode/extensions/vknabel.vscode-swiftlint-1.7.2/out/lint.js:5:58)
	at processTicksAndRejections (node:internal/process/task_queues:96:5)
os: darwin
```

**Comment by vknabel** [comment]
Hi. It seems like your Swiftlint crashes and stops printing the result.
Does this happen repeatedly?
Does this happen within an open source project?
Can you reproduce this crash by manually Running Swiftlint on the command line?

**Comment by talktosalvador** [comment]
Hi, I no longer have the error and I can't reproduce it again.
Just add this parameter to automatically correct when saving changes, is it the right way?
![image](https://user-images.githubusercontent.com/81868766/204579716-b3db5f20-c8b2-4fae-85cd-9c4a51b034b7.png)

**Comment by vknabel** [comment]
Has been fixed in [1.7.3](https://github.com/vknabel/vscode-swiftlint/releases/tag/1.7.3).
If your issue has not been fixed yet, feel free to reopen.

>To automatically fix all issues within a document on save, add the following to your `.vscode/settings.json`:
>
> ```json
> {
>  "editor.codeActionsOnSave": {
>    "source.fixAll": true
>  }
> }
> ```
>
> Regarding the error you received: this should be fixed in the next release.

See https://github.com/vknabel/vscode-swiftlint/issues/54#issuecomment-1346249577

**Comment by talktosalvador** [comment]
hero!, works perfect
